lib/assessment.js

```diff
@@ -13,14 +22,24 @@ var min = function min() {
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
-
+  var arg = arguments;
+  var total = 0;
+  //not entering the if statement. Sigh.
+  if(arg){
+    for (var i = 0, length = arg.lenth; i < length; i++) {
+      total += arg[i];
+    }
+  } return total;
```

Comment:
Incidentally, it's not really necessary to nest all your code in an if - what would be simpler is just handling special cases up front and returning early.
e.g.
```javascript
var sum = function sum() {
  if (arguments.length === 0) {
   return 0;
  }
  var total = 0;
  for (var i = 0; i < arguments.length; i++) {
   total += arguments[i];
  }
  return total;
};
```
In this case, though, it's actually not even necessary to do something like that, since if there are no arguments, the `for` loop never runs, and so we just return 0.